README.md

```diff
@@ -74,6 +75,16 @@ Questa pagina è attualmente curata dalla [community di PAC](https://www.faceboo
 * GDG Gela - [<img src="media/web.png" width="18">](http://www.gdggela.org/)
 * CoderDojo Gela - [<img src="media/web.png" width="18">](http://www.coderdojogela.it/) [<img src="media/fb-page.png" width="18">](https://www.facebook.com/coderdojogela)
 
+### Coworking
+
+#### Catania
+
+* Vulcanìc - [<img src="media/web.png" width="18">](http://www.vulcanic.it/)
```

Comment:
I am actually not sure on how to deal with duplicates across the list.
If you think Vulcanis is more of co-working place rather than an Incubator, then move it.